src/Components/CollectionsRail/CollectionsRail.tsx

```diff
@@ -21,38 +25,52 @@ const RailsWrapper = styled(Flex)`
   max-width: 1250px;
   margin: 0 auto;
   flex-direction: column;
+
   ${pMedia.xl`
-    padding: 40px;
-  `};
-  ${pMedia.lg`
-    padding: 40px;
+    padding: ${space(4)}px;
   `};
-  ${pMedia.xs`
-    padding: 40px 30px;
+  ${pMedia.sm`
+    padding: ${space(4)}px ${space(2)}px;
```

Comment:
Made some minor updates here to remove extra declaration, and match padding to `RelatedArticlesCanvas`